gdx/src/com/badlogic/gdx/graphics/g3d/shadow/filter/NoLightFilter.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.graphics.g3d.shadow.filter;
+
+import com.badlogic.gdx.graphics.Camera;
+import com.badlogic.gdx.graphics.g3d.environment.BaseLight;
+
+public class NoLightFilter implements LightFilter {
+
+	@Override
+	public boolean filter (int n, BaseLight light, Camera camera) {
+		return true;
```

Comment:
`LightFilter.filter` docs state "Return true if the light has to be used for rendering". I would expect a `NoLightFilter` to return false then.